app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
+            shopify_refund = ShopifyAPI::Refund.find(shopify_refund_id, params: { order_id: shopify_order_id })
+            Spree::Retail::Shopify::GenerateRefundOrder.new(shopify_refund).process
```

Comment:
Let's wait until it's motivated by pain. I think at some point we'll need to attack the idea of importing a refund into Solidus, but that day is not today!